rt-forecast/format-rt-fn.R

```diff
@@ -74,28 +76,38 @@ format_rt_forecast <- function(loc = NULL, loc_name = NULL,
 
 # Get estimates ---------------------------------------------------------------
 
+  file_loc <- paste0(loc, "/", forecast_date)
   
-  ## Check if latest folder exists; if not, return NULL
-  if (!dir.exists(paste0(loc, "/latest/")) ){
-    return(NULL)
+  
```

Comment:
Turned this into an optional argument as I'm worried this could cause mix ups in the standard submission formatting.

Reply:
don't think it would. either you run it on the submission date and then the folder should be present - or you do it later then that and then it just catches that. But anyway, happy to keep it if you prefer that